src/style.css

```diff
@@ -1 +1,100 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.form {
+  display: flex;
+  justify-content: center; /* Центрування форми по горизонталі */
+  align-items: center;
+  width: 100%; /* Ширина для flex-контейнера */
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here, you've added a margin-top to `.form`, which might lead to margin collapse issues if you're not careful.